src/Antd_Grid.re

```diff
@@ -57,10 +57,41 @@ module Row = {
       ~gutter: 'a=?,
       ~style: ReactDOMRe.Style.t=?,
       ~prefixCls: string=?,
-      ~children: React.element=?
+      ~children: React.element=?,
+      unit
     ) =>
-    React.element =
-    "antd/lib/grid/row";
+    _ =
+    "";
+
+  [@bs.module]
+  external reactComponent: React.component('a) = "antd/lib/grid/row";
+
+  [@react.component]
+  let make =
+      (
+        ~className: option(string)=?,
+        ~_type: option(string)=?,
+        ~align: option(string)=?,
+        ~justify: option(string)=?,
+        ~gutter: option('a)=?,
+        ~style: option(ReactDOMRe.Style.t)=?,
+        ~prefixCls: option(string)=?,
+        ~children: option(React.element)=?,
+      ) =>
+    React.createElement(
+      reactComponent,
+      makePropsGridRow(
```

Comment:
Sticking to composition across different files union files should be passed as polymorphic variant type like so
```
~_type=Belt.Option.map(_type, rowTypeToJs)
~align=Belt.Option.ma(align,rowAlignToJs)
~justify=Belt.Option.map(justify,rowJustifyToJs)
```